src/src/AdminBundle/Resources/views/Default/index.html.twig

```diff
@@ -1,67 +1,44 @@
-{% extends "@AdminBundle/Resources/views/base.html.twig" %}
+{% extends "@AdminBundle/Resources/views/base_admin.html.twig" %}
 
```

Comment:
tento file vobec potrebujeme? na co?

Reply:
well, to je uvodna stranka po prihlaseni, ta sa casom prerobi na `Event`

Reply:
A nepremenujeme to uz teraz z Default na Events?

Reply:
oki, moze byt